src/test/java/org/folio/consortia/support/extension/impl/KafkaContainerExtension.java

```diff
@@ -0,0 +1,30 @@
+package org.folio.consortia.support.extension.impl;
+
+import static org.testcontainers.utility.DockerImageName.parse;
+
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.testcontainers.containers.KafkaContainer;
+import org.testcontainers.utility.DockerImageName;
+
+public class KafkaContainerExtension implements BeforeAllCallback, AfterAllCallback {
+  private static final String SPRING_PROPERTY_NAME = "spring.kafka.bootstrap-servers";
+  private static final DockerImageName KAFKA_IMAGE = parse("confluentinc/cp-kafka:5.5.3");
```

Comment:
Lets use newer version as in mod-users: [Link](https://github.com/folio-org/mod-users/pull/286/files#diff-ee5f769c79d2bb880ed12399e98ecb357ff61e5e7d0d00e0b451d30b88a02cd8R39 )

Reply:
Sure. I was thinking this old version was chosen intentionally